models/model.js

```diff
@@ -1,9 +1,19 @@
 const db = require('../db/connection')
+const articles = require('../db/data/test-data/articles');
 
 function getTopics() {
     return db.query('SELECT * FROM topics').then((result) => {
         return result.rows;
     });
 }
 
-module.exports = { getTopics };
\ No newline at end of file
+function getArticle(id) {
+    const notFoundMsg = { status: 404, message: '404: Not found'};
+    const badReqMsg = { status: 400, message: '400: Bad request'};
```

Comment:
you're not using these two variables anywhere (and in my opinion I don't think they're any clearer than just sending the status and message manually as you have done below)